### PR TITLE
[MODULAR] Icebox Permabrig Revamped

### PR DESCRIPTION
**Pull request by MassacreDemigirl**
## About The Pull Request
Removes most of the sex dungeons from the perma, moves the Prisoner Transfer and Witness Cell a bit to accommodate a full maintenance tunnel and adds a stripped down LustWish and a pool for good prisoners

## How This Contributes To The Skyrat Roleplay Experience
Makes the Icebox Prison more in line with a real perma brig while keeping the added amenities by letting the Security/Warden/CO decide. Giving the prisoners incentive to be good in order to get access to the lounge. Making the lustwish not vend contraband (for prisoners) makes it more acceptable as well.

## Media
newbrig

## Changelog

:cl:
add: Icebox Permabrig's construction is finally done! A new prisoner lounge for good prisoners and a full size maint tunnel has been added! As well as a NanoTrasen approved LustWish to boot!
/:cl:

### CODE REVIEW

File: _maps/map_files/IceBoxStation/IcemoonUnderground_Above_skyrat.dmm [hunk 1]
```diff
@@ -16272,6 +16422,17 @@
 /obj/item/crowbar,
 /turf/open/floor/iron,
 /area/service/chapel)
+"VX" = (
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4,
+/obj/machinery/vending/dorms{
+	desc = "A vending machine with various toys. Not for the faint of heart. This one seems to have a reduced inventory.";
+	name = "NanoTrasen Approved LustWish";
```
Comment (Zonespace27) [inline _maps/map_files/IceBoxStation/IcemoonUnderground_Above_skyrat.dmm]: May want to make this its own type to allow for use of it outside this map without copious varediting

**Comment by MassacreDemigirl** [comment]
image
I fixed this stupid table

File: _maps/map_files/IceBoxStation/IcemoonUnderground_Above_skyrat.dmm (hunk 1 above)
Comment (MassacreDemigirl) [reply _maps/map_files/IceBoxStation/IcemoonUnderground_Above_skyrat.dmm]: Hm sure makes sense

**Comment by KathrinBailey** [comment]
Nah, not going with this. The justification for this PR is far too weak.
That closed shutter will just just remain closed 99% of shifts.
Even then the seccie lobby group are going to complain about it looking too fancy and claim it's now a seven star hotel.
I'm also not liking how many tiles are getting vented from the maint tunnels by a seccie just walking around, and there's no good reason for them to even be there in the first place.

**Pull request closed without merging**